framework/math/vector_ghost_communicator/vector_ghost_communicator.cc

```diff
@@ -71,7 +71,7 @@ VectorGhostCommunicator::MakeCachedParallelData()
   // MPI utility MapAllToAll in Chi-Tech accomplishes this task,
   // returning a mapping of processes to the global ids that this
   // process needs to send.
-  std::map<int, std::vector<int64_t>> send_map = MapAllToAll(recv_map, MPI_INT64_T, comm_);
```

Comment:
how is MPI_INT64_T incorporated in the new mpi wrappers?

Reply:
There is no direct mapping for `MPI_INT64_T`. But `long long int` should be that on a 64bit architecture. I can add the `intN_t` types, since MPI has mapping for them.

Reply:
Never mind this. If I add those new types, compiler sees them as equivalent and reports a redefinition error. So this is captured correctly...